<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityStatusWidget</class>
 <widget class="QWidget" name="EntityStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="status_image_label">
       <property name="text">
        <string>Status Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="entity_title_label">
       <property name="text">
        <string>Entity Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_message_label">
       <property name="text">
        <string>Status Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
